quickstarts/lighttpd/config.yml

```
@@ -0,0 +1,40 @@
+
+slug: lighttpd
+description: |
+  ## Comprehensive monitoring quickstart for Lighttpd
+
+  Lighttpd is an open-source web server designed for high-performance situations while staying standards-compliant, secure and adaptable.
```

```suggestion
  Lighttpd is an open-source web server designed for high-performance situations while staying standards-compliant, secure, and adaptable.
```

This is a great intro sentence! We prefer the Oxford (or serial) comma though :)